src/get-started/install/_path-linux-chromeos.md

```diff
@@ -1,90 +1,76 @@
-use the following instructions.
-
-The steps for modifying the `PATH` variable
-_all_ subsequent terminal sessions are machine-specific.
-Typically, you add a line to a shell script file that
-executes whenever you open a new window. For example:
-
- 1. Determine the path of your clone of the Flutter SDK.
-    You need this in Step 3.
- 2. Open (or create) the `rc` file for your shell.
-    For example, Linux uses the Bash shell by default,
-    so edit `$HOME/.bashrc`.
-    If you are using a different shell, the file path
-    and filename will be different on your machine.
- 3. Add the following line, changing the
-    `[PATH_OF_FLUTTER_GIT_DIRECTORY]` to
-    the path of your copy of the Flutter SDK:
-
-    ```terminal
-    $ export PATH="$PATH:[PATH_OF_FLUTTER_GIT_DIRECTORY]/bin"
-    ```
-
- 4. Run `source $HOME/.<rc file>`
-    to refresh the current window,
-    or open a new terminal window to
-    automatically source the file.
- 5. Verify that the `flutter/bin` directory
-    is now in your PATH by running:
-
-    ```terminal
-    $ echo $PATH
-    ```
-
-    Verify that the `flutter` command is available by running:
-
-    ```terminal
-    $ which flutter
-    ```
+Whether you've installed Flutter using `snapd` or manually, you'll need to add the Flutter SDK to your PATH. The procedure varies depending on your shell and whether you want to add it to the PATH temporarily (for the current terminal session only) or permanently.
 
 {% include docs/dart-tool.md %}
 
-### Update the path directly
+#### Temporarily update your path
+
+You can update your PATH variable for the current session at the command line:
+
+```terminal
+$ export PATH="$PATH:[PATH_TO_FLUTTER_GIT_DIRECTORY]/flutter/bin"
+```
+
+In this command, replace `[PATH_TO_FLUTTER_GIT_DIRECTORY]` with your actual Flutter SDK path.
+
+#### Permanently update your path
+
+To permanently add Flutter to your path for _any_ terminal session, follow these steps:
+
+1. Determine the path of your Flutter SDK installation. You'll need this in Step 3.
+2. Open (or create) the `rc` file for your shell. This file runs whenever you open a new terminal window. If you're using the Bash shell, for example, edit `$HOME/.bashrc`.
+3. Add this line, replacing `[PATH_TO_FLUTTER_GIT_DIRECTORY]` with your Flutter SDK path:
 
-In some cases, your distribution might not permanently acquire
-the path when using the previous directions. When this occurs,
-you can change the environment variables file directly.
-These instructions require administrator privileges:
+```terminal
+export PATH="$PATH:`pwd`/flutter/bin"
+```
+
+4. Run `source $HOME/.<rc file>` to refresh the current window, or open a new terminal window to automatically source the file.
+5. Verify that the `flutter/bin` directory is now in your PATH by running:
+
+```terminal
+$ echo $PATH
+```
+
+Verify that the `flutter` command is available by running:
+
+```terminal
+$ which flutter
+```
+{% include docs/dart-tool.md %}
```

Comment:
**issues:**

1. Start any ordered list item with `1.`. This enables automatic numbering.
2. Indent code blocks under the list item.
3. Provide a response to all terminal commands.
4. Format CLI commands per [docopt](http://docopt.org/).

````suggestion
  ```terminal
  $ echo 'export PATH="$PATH:<path_to_flutter_directory>/flutter/bin"' >> $HOME/.bashrc
  ```

1. Reload the current shell profile.

  ```terminal
  source $HOME/.<rc file>
  ```

1. Verify that the `flutter/bin` directory exists in your PATH.

  ```terminal
  $ echo $PATH
  ```

  Response should resemble:
  
  ```terminal
  /usr/<example>/dev/flutter/bin:/usr/local/git/git-google/bin:/usr/local/git/current/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:
  ```

1. Verify that you can now use the `flutter` command.

  ```terminal
  $ which flutter
  ```

  Response should resemble:
  
  ```
  /usr/<example>/dev/flutter/bin/flutter
  ```

{% include docs/dart-tool.md %}
````